src/test/java/io/github/mysterix5/PasswordValidatorTest.java

```diff
@@ -1,15 +1,145 @@
 package io.github.mysterix5;
```

Comment:
In deiner Testklasse befindet sich sehr viel Logik, wie z.B. die Ermittlung welche Testpasswörter valide sind und welche nicht. Ich würde versuchen Tests so zu schreiben, dass man sie einfach von oben nach unten lesen kann ohne den Kontext der Testmethode zu verlassen. Ich empfehle folgende Lektüre: https://mtlynch.io/good-developers-bad-tests/